docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -2,6 +2,15 @@
 Pulp 2.11 Release Notes
 =======================
 
+.. note::
```

Comment:
I think notes are easy to ignore, so this should be a section, not a note, with heading "EL6 Support Changes" or something.